codespell_lib/data/dictionary.txt

```diff
@@ -7874,6 +7904,13 @@ crasheed->crashed
 crashees->crashes
 crashess->crashes
 crashs->crashes
+cration->creation, nation,
+crationalism->nationalism
+crationalist->nationalist
+crationalists->nationalists
+crationist->creationist
+crationists->creationists
+crations->creations, nations,
```

Comment:
What about suggesting `ration`, `rationalism`, `rationalist`, `rationalists`, `rations` ?

Reply:
Yeah, those are really great alternatives — a little suprised I didn't think of them myself 😆